client/allocrunner/taskrunner/envoy_bootstrap_hook.go

```diff
@@ -402,6 +410,10 @@ func (e envoyBootstrapArgs) args() []string {
 		arguments = append(arguments, "-gateway", e.gateway)
 	}
 
+	if v := e.proxyID; v != "" {
+		arguments = append(arguments, "-proxy-id", e.proxyID)
```

Comment:
should this append `v` instead? 
```suggestion
		arguments = append(arguments, "-proxy-id", v)
```